### PR TITLE
hm2.2solution

### PR DESCRIPTION
**Pull request by voloshynamaryna11**

### CODE REVIEW

File: src/main/java/core/basesyntax/Bulldozer.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+class Bulldozer extends Machine {
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/Bulldozer.java:3]:
```suggestion
public class Bulldozer extends Machine {
```

File: src/main/java/core/basesyntax/Excavator.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+class Excavator extends Machine {
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/Excavator.java:3]: add access modifier

File: src/main/java/core/basesyntax/MainApp.java [hunk 3]
```diff
@@ -12,5 +12,23 @@
...
+        excavator.stopWork();
+    }
 }
+
+
+
+
+
+
+
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/MainApp.java:33]: remove all redundant lines at the end of the file

File: src/main/java/core/basesyntax/Track.java [hunk 4]
```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+class Track extends Machine {
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/Track.java:3]: add access modifier

File: src/main/java/core/basesyntax/Track.java [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+    @Override
+    public void doWork() {
+        System.out.println("Track start working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Track stop working");
+    }
+
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/Track.java:14]: let's remove empty line

File: src/main/java/core/basesyntax/Bulldozer.java (hunk 1 above)
Comment (voloshynamaryna11) [reply src/main/java/core/basesyntax/Bulldozer.java]: done

File: src/main/java/core/basesyntax/Excavator.java (hunk 2 above)
Comment (voloshynamaryna11) [reply src/main/java/core/basesyntax/Excavator.java]: done

File: src/main/java/core/basesyntax/MainApp.java (hunk 3 above)
Comment (voloshynamaryna11) [reply src/main/java/core/basesyntax/MainApp.java]: done

File: src/main/java/core/basesyntax/Track.java (hunk 4 above)
Comment (voloshynamaryna11) [reply src/main/java/core/basesyntax/Track.java]: done

File: src/main/java/core/basesyntax/Track.java (hunk 5 above)
Comment (voloshynamaryna11) [reply src/main/java/core/basesyntax/Track.java]: done

**Pull request closed without merging**